src/chrome/content/rules/Steamrep.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Steamrep.com">
+  <target host="steamrep.com" />
+  <target host="www.steamrep.com" />
+
+  <rule from="^http://(www\.)?steamrep\.com/" to="https://www.steamrep.com/" />
```

Comment:
I think @semenko's suggestion is better: Since only steamrep.com works over HTTPS, both steamrep.com and www.steamrep.com should redirect to the HTTPS version of steamrep.com.